### PR TITLE
chore: add missing translation tags

### PR DESCRIPTION
**Pull request by nicholasyao974**
# Description

copilot:summary

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/35958863/todos/7014002206)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

copilot:walkthrough

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/RadioOption/RadioOption.tsx [hunk 1]
```diff
@@ -31,16 +33,16 @@ export function RadioOption({
...
-        name="Action"
-        value={
+        name={t('Action')}
+        value={t(
           actions.find((act) => act.value === action?.__typename)?.label ??
-          'None'
-        }
-        description="Action"
+            'None'
+        )}
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/RadioOption/RadioOption.tsx:40]: I think the variable isn't being interpolated here might have to do the {{ insert variable name here }}. And we're wrapping the logic within the t function. Lets get away from it if we can

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/TextResponse/TextResponse.tsx [hunk 2]
```diff
@@ -32,13 +34,13 @@ export function TextResponse({
       <Attribute
         id={`${id}-text-field-options`}
         icon={<TextInput1Icon />}
-        name="Feedback"
-        value={label}
-        description="Feedback Properties"
+        name={t('Feedback')}
+        value={t(label)}
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/TextResponse/TextResponse.tsx:38]: not going to work

File: apps/journeys-admin/src/components/AccessDialog/AddUserSection/AddUserSection.tsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ export function AddUserSection({
           }}
           onClick={handleClick}
         >
-          <Typography variant="body2">{selectedInviteMethod}</Typography>
+          <Typography variant="body2">{t(selectedInviteMethod)}</Typography>
```
[the remark this answers is not in the document]
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/AccessDialog/AddUserSection/AddUserSection.tsx:89]: It's translating for me. We don't have a translation for the "Link" text yet so it defaults to English
image

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/Action/Action.tsx [hunk 4]
```diff
@@ -192,7 +197,7 @@ export function Action(): ReactElement {
                     nextStep == null && action.value === 'NavigateAction'
                   }
                 >
-                  {action.label}
+                  {t(action.label)}
```
[the remark this answers is not in the document]
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/Action/Action.tsx:200]: It does need to be wrapped. The setup we have of translating the exported stuff is kinda jank where wrapping them in t tags at the place where they're defined only makes it so running extract-translations will add them to the locale file. They don't actually get translated at runtime, so any variables that can take those values need to be wrapped as well to translate them. If you can find a better way please let me know!

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/Action/Action.tsx [hunk 5]
```diff
@@ -49,26 +50,30 @@ export const ACTION_DELETE = gql`
     }
   }
 `
+init({
```
[the remark this answers is not in the document]
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/Action/Action.tsx]: As I said below yeah this is kind of weird. I have just realised that the init function seems to be unnecessary though.

File: libs/locales/en/apps-journeys-admin.json [hunk 6]
```diff
@@ -420,6 +605,7 @@
   "Unarchive Templates": "Unarchive Templates",
   "Are you sure you would like to unarchive all archived templates immediately?": "Are you sure you would like to unarchive all archived templates immediately?",
   "Are you sure you would like to trash all archived templates immediately?": "Are you sure you would like to trash all archived templates immediately?",
+  " - {{ description }}": " - {{ description }}",
```
[the remark this answers is not in the document]
Comment (nicholasyao974) [reply libs/locales/en/apps-journeys-admin.json:611]: Yeah it's preceded by a date - so I think it'll need to be translated so it displays properly on RTL
image

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Button/Button.tsx [hunk 7]
```diff
@@ -79,13 +81,13 @@ export function Button({
       <Attribute
         id={`${id}-button-color`}
         icon={<ColorDisplayIcon color={buttonColor} />}
-        name="Color"
-        value={capitalize(buttonColor?.toString() ?? ButtonColor.primary)}
-        description="Background Color"
+        name={t('Color')}
+        value={t(capitalize(buttonColor?.toString() ?? ButtonColor.primary))}
```
[the remark this answers is not in the document]
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Button/Button.tsx:85]: It works :)

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/TextResponse/TextResponse.tsx (hunk 2 above)
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/TextResponse/TextResponse.tsx:38]: This one is also working for me

image

File: apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/RadioOption/RadioOption.tsx (hunk 1 above)
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/RadioOption/RadioOption.tsx:40]: It's working properly for me. What does it look like on your end?
Screenshot 2024-02-07 132926

Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/RadioOption/RadioOption.tsx:40]: I can take the logic out of the t tag though

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx [hunk 8]
```diff
@@ -58,19 +60,21 @@ export function JourneyCardInfo({
...
+                    fontWeight: 700,
+                    minWidth: '56px'
+                  }}
+                >
+                  {t('{{ usersRequestingAccess }}', { usersRequestingAccess })}
+                </Typography>
+                <Typography variant="body2" noWrap>
+                  {t(`requested editing rights for your journey`)}
+                </Typography>
+              </Trans>
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx:77]: Either use the Trans component or just the t function. Currently with the Trans component wrapped it's exporting the t function quite literally
Screenshot 2024-02-09 at 09 39 21

**Review by GeronimoJohn — DISMISSED** [review]

**Review by GeronimoJohn — DISMISSED** [review]

**Review by edmonday — APPROVED** [review]
nice

**Review by edmonday — APPROVED** [review]

**Pull request merged**